test/swaggerToSDKTests.ts

```diff
@@ -115,7 +115,8 @@ describe("SwaggerToSDK", function () {
         ]);
       });
 
-      it("when diff_url returns non-empty body", async function () {
+      it("end-to-end", async function () {
+        this.timeout(600000);
```

Comment:
10 minutes seems excessive?

Reply:
This won't be a permanent test. The test currently takes around 4 minutes to run, but I'll be breaking it into smaller pieces as I go along.